packages/medusa/src/api/routes/store/carts/decorate-line-items.ts

```diff
@@ -0,0 +1,27 @@
+import { Request } from "express"
+import { TotalsService } from "../../../../services"
+import { Cart, LineItem } from "../../../../models"
+
+export const decorateLineItems = async (
```

Comment:
Yes had the same thought - but wanted to wait for @adrien2p before trying to incorporate anything, so took the naive approach.

Reply:
Yes indeed we can use the interceptor pattern which allows to manipulate the input data as well as the output data.

I have an architecture and a first implementation in the head, but not yet the time to introduce that. When i've finish the auth strategies pr write up, it could be my next DD? @olivermrbl @srindom